core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -172,6 +175,13 @@ public void run() {
     }
   }
 
+  private void createVersionMetric(MetricsSystem metricsSystem) {
+    metricsSystem
+        .createLabelledGauge(
+            Web3SignerMetricCategory.WEB3SIGNER, "version", "Release information", "version")
```

Comment:
Not sure why we need a labelled gauge. But we are taking that approach then then I think the metric should be release with the labelled metric of version.
i.e. 
`metricsSystem
        .createLabelledGauge(
            StandardMetricCategory.PROCESS, "release", "Release information", "version")
        .labels(() -> 1, ApplicationInfo.version();`